app/views/organizations/staff/shared/_attachment_form.html.erb

```diff
@@ -16,19 +16,18 @@
                           direct_upload: true,
                           class: "custom-attachments",
                           hide_label: true %>
-          <small class="form-text text-muted"> 
+          <small class="form-text text-muted">
             <% if attachment_type == 'files' %>
               Files must be .pdf, .png, or .jpeg under 2MB.
             <% elsif attachment_type == 'images' %>
-              Images must be .jpg or .png under 1MB. 
-            <%end%>
+              Images must be .jpg or .png under 1MB.
+            <% end %>
           </small>
         </div>
         <div class="ps-2">
           <%= form.submit t("general.attach"), class: "btn btn-outline-success" %>
         </div>
       </div>
     <% end %>
-    <%= turbo_frame_tag "results" %>
```

Comment:
Moved to the external form index. This partial is used in multiple locations and the results are really only need for the csv import.